steps/autify-upload/0.0.1/step.yml

```diff
@@ -0,0 +1,69 @@
+title: |
+  Autify Upload
+summary: |
+  Uploads your build to Autify.
+description: |-
+  Uploads you build `.app` to [Autify](https://autify.com/mobile) using Autify's API. You can upload the iOS build with the Step.
+
+  This Step, however, does NOT generate your build: to create a `.app` file, you need the right step. For example, `Xcode build for simulator` Step or any other Step can generate a build. Add task URL
+website: https://github.com/autifyhq/bitrise-steps-autify-upload
+source_code_url: https://github.com/autifyhq/bitrise-steps-autify-upload
+support_url: https://github.com/autifyhq/bitrise-steps-autify-upload/issues
+published_at: 2021-06-02T05:10:23.914108+09:00
+source:
+  git: https://github.com/autifyhq/bitrise-step-autify-upload.git
+  commit: 5d9bc7aba74c96d11c5cd4098b82e1537350f5c9
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+- android
```

Comment:
It contradicts with a description.

Reply:
thanks.
I removed this tag.

Reply:
@koral-- Could you check it out again. Thanks in advance